tests/network/vmi_multus.go

```diff
@@ -332,26 +353,43 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 		})
 
 		Context("VirtualMachineInstance with Linux bridge plugin interface", func() {
-			table.DescribeTable("should be able to ping between two vms", func(interfaces []v1.Interface, networks []v1.Network, ifaceName string) {
+			table.DescribeTable("should be able to ping between two vms", func(interfaces []v1.Interface, networks []v1.Network, ifaceName, cidr1, cidr2 string) {
+				if cidr2 == "dhcp" {
+					ipam := "\\\"type\\\": \\\"host-local\\\", \\\"subnet\\\": \\\"10.1.1.0/24\\\", \\\"rangeStart\\\": \\\"10.1.1.2\\\", \\\"rangeEnd\\\": \\\"10.1.1.3\\\""
+					Expect(createBridgeNetworkAttachementDefinition("linux-bridge-net-ipam", "0", ipam))
+				}
+
 				vmiOne := createVMIOnNode(interfaces, networks)
 				vmiTwo := createVMIOnNode(interfaces, networks)
 
 				tests.WaitUntilVMIReady(vmiOne, console.LoginToAlpine)
 				tests.WaitUntilVMIReady(vmiTwo, console.LoginToAlpine)
 
-				Expect(configInterface(vmiOne, ifaceName, "10.1.1.1/24")).To(Succeed())
+				Expect(configureAlpineInterface(vmiOne, ifaceName, cidr1)).To(Succeed())
 				By(fmt.Sprintf("checking virtual machine interface %s state", ifaceName))
 				Expect(checkInterface(vmiOne, ifaceName)).To(Succeed())
 
-				Expect(configInterface(vmiTwo, ifaceName, "10.1.1.2/24")).To(Succeed())
+				Expect(configureAlpineInterface(vmiTwo, ifaceName, cidr2)).To(Succeed())
 				By(fmt.Sprintf("checking virtual machine interface %s state", ifaceName))
 				Expect(checkInterface(vmiTwo, ifaceName)).To(Succeed())
 
-				By("ping between virtual machines")
-				Expect(libnet.PingFromVMConsole(vmiOne, "10.1.1.2")).To(Succeed())
+
+				if cidr2 == "dhcp" {
+					By("ping between virtual machines")
+					// with DHCP we don't know which VMI got which address from the range, so we ping both
+					Expect(libnet.PingFromVMConsole(vmiOne, "10.1.1.2")).To(Succeed())
+					Expect(libnet.PingFromVMConsole(vmiOne, "10.1.1.3")).To(Succeed())
```

Comment:
Where did these come from ? 

You'll need to come up with a function to retrieve an IP address from the range assigned by IPAM: https://github.com/kubevirt/kubevirt/pull/5826/commits/0383798120d1346dff8fa7cd87121b3621c28f82#diff-da14ca97e77db6bb275427ac69187a539c58fc297e64519628e5385f9f411c5fR358

Reply:
I decided to remove the range and retrieve the address from the pod with this [function](https://github.com/kubevirt/kubevirt/compare/716efb2f60a6e11b8da900adcc6c3ea080a58e77..3e98959bc57018af7cb77a6cf59adb0b399b3c1e#diff-da14ca97e77db6bb275427ac69187a539c58fc297e64519628e5385f9f411c5fR356-R375)